packages/wellknown-did-issuer/src/migrations/postgres/1661165115000-CreateWellknownDidIssuer.ts

```diff
@@ -0,0 +1,19 @@
+import { MigrationInterface, QueryRunner } from 'typeorm'
+
+export class CreateWellknownDidIssuer1661165115000 implements MigrationInterface {
+  name = 'CreateWellknownDidIssuer1661165115000'
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `CREATE TABLE "DidConfigurationResource" ("origin" character NOT NULL, "context" character varying NOT NULL, "created_at" TIMESTAMP NOT NULL DEFAULT now(), "last_updated_at" TIMESTAMP NOT NULL DEFAULT now(), CONSTRAINT "PK_Origin" PRIMARY KEY ("origin"))`
```

Comment:
Is there a reason why origin isn't a varchar?

Reply:
bad copy paste, fixed